### PR TITLE
51.6.0 (latest-rc) RNs

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

**Review by RodEsp — APPROVED** [review]

File: releasenotes/README.md [hunk 1]
```diff
@@ -12,10 +12,20 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
...
 [Click here for the v50 release notes.](./v50.md)
 
-## March 25, 2021
+## April 1, 2021
 
 These changes are in the release candidate plug-in (`salesforcedx@latest-rc`). We plan to include these changes in the next official release. This list isn't final and is subject to change.
 
+* NEW: Get a list of files in your project that the `force:source:*` commands ignore with the new `force:source:ignored:list` command. The command refers to the `.forceignore` file to determine the list of ignored files. Run the command without any parameters to get a list of all the files in all package directories that are ignored. Use the `--sourcepath` parameter to limit the check to a specific file or directory. If you specify a directory, the command checks all subdirectories recursively. For example, run this command to check if a particular file is ignored:
+
+    `sfdx force:source:ignored:list --sourcepath=mypackage.xml`
```
Comment (ekapner) [inline releasenotes/README.md:21]: I might even use just package.xml - cuz that's a file I suspect someone would def want to ignore.

**Review by ekapner — COMMENTED** [review]
Looks like this one was already merged - and my comment is definitely not something that HAS to be addressed.

File: releasenotes/README.md (hunk 1 above)
Comment (jshackell-sfdc) [reply releasenotes/README.md:21]: Yah, i like it.

**Comment by jshackell-sfdc** [comment]
I did a direct update to main with your suggestion, Emily. I made the exec decision that it doens't need a PR.

**Pull request merged**